agent/util-scripts/pbench-tool-meister-start

```diff
@@ -340,6 +342,18 @@ def main(argv):
         ), f"bad channel: {resp!r}"
         assert resp["data"] == 1, f"bad data: {resp!r}"
 
+    # 2.5. Add tool metadata json to redis
+    try:
+        tm_start_path = Path(
+            Path(__file__).resolve(), "..", ".."
+        ).resolve(strict=True)
+    except FileNotFoundError:
+        raise Exception(f"missing {tm_start_path}")
```

Comment:
If we can't resolve the location of the installation directory, it would seem that `tm_start_path` would be defined.  Perhaps we could log the exception as an error, like: `logger.error("Unable to determine the proper installation directory, expected directory %s not found", install_dir)`?